.github/workflows/build.yml

```diff
@@ -35,6 +35,7 @@ jobs:
     needs: gradleValidation
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         product: [ "IC-2020.2", "IC-2021.1", "IC-2022.1" ]
```

Comment:
should we add 2022.2?

Reply:
I can do that in a followup PR, if that's okay with you.

Reply:
Related PR: https://github.com/amazon-ion/ion-intellij-plugin/pull/41